skopt/tests/test_common.py

```diff
@@ -306,7 +306,26 @@ def test_repeated_x():
         assert_warns_message(
             UserWarning, "has been evaluated at", minimizer, lambda x: x[0],
             dimensions=[[0, 1]], x0=[[0], [1]], n_random_starts=0, n_calls=3)
+
         assert_warns_message(
             UserWarning, "has been evaluated at", minimizer, bench4,
-            dimensions=[("0", "1")], x0=["0", "1"], n_calls=3,
+            dimensions=[("0", "1")], x0=[["0"], ["1"]], n_calls=3,
```

Comment:
Maybe we should also add some protection in the optimizer to catch this kind of mistake (I think the `assert` statements would cover this, but didn't test it.